src/pages/ReimbursementAccount/ReimbursementAccountPage.tsx

```diff
@@ -289,14 +282,13 @@ function ReimbursementAccountPage({route, policy}: ReimbursementAccountPageProps
             Navigation.setParams({stepToOpen: getRouteForCurrentStep(currentStep)});
         },
         // eslint-disable-next-line react-compiler/react-compiler, react-hooks/exhaustive-deps
-        [isOffline, reimbursementAccount, route, hasACHDataBeenLoaded, shouldShowContinueSetupButton],
+        [isOffline, reimbursementAccount, route, hasACHDataBeenLoaded, shouldShowContinueSetupButton, policyIDParam],
```

Comment:
I think this is unnecessary. When policyIDParam change this component will be rendered for sure (because policyIDParam change means that the route change). Please correct me if I am wrong

Reply:
yep and we no longer use it here - removed!